reference/docs-conceptual/install/Installing-PowerShell-on-Linux.md

```diff
@@ -1,943 +1,96 @@
-| Ubuntu 18.04       | libc6, libgcc1, libgssapi-krb5-2, liblttng-ust0, libstdc++6, <br> libcurl3, libunwind8, libuuid1, zlib1g, libssl1.0.0, libicu60 |
-| Debian 8 (Jessie)  | libc6, libgcc1, libgssapi-krb5-2, liblttng-ust0, libstdc++6, <br> libcurl3, libunwind8, libuuid1, zlib1g, libssl1.0.0, libicu52 |
-| Debian 9 (Stretch) | libc6, libgcc1, libgssapi-krb5-2, liblttng-ust0, libstdc++6, <br> libcurl3, libunwind8, libuuid1, zlib1g, libssl1.0.2, libicu57 |
-| CentOS 7 <br> Oracle Linux 7 <br> RHEL 7 | libunwind, libcurl, openssl-libs, libicu |
-| openSUSE 42.3 | libcurl4, libopenssl1_0_0, libicu52_1 |
-| openSUSE Leap 15 | libcurl4, libopenssl1_0_0, libicu60_2 |
-| Fedora 27 <br> Fedora 28 | libunwind, libcurl, openssl-libs, libicu, compat-openssl10 |
-
-To deploy PowerShell binaries on Linux distributions that aren't officially supported, you need to
-install the necessary dependencies for the target OS in separate steps. For example, our
-[Amazon Linux dockerfile][amazon-dockerfile] installs dependencies first, and then extracts the
-Linux `tar.gz` archive.
+Fedora uses DNF as its package manager.
 
-[amazon-dockerfile]: https://github.com/PowerShell/PowerShell-Docker/blob/master/release/community-stable/amazonlinux/docker/Dockerfile
+[!INCLUDE [Fedora support](../../includes/fedora-support.md)]
 
-### Installation - Binary Archives
+For more information, see [Install .NET on Fedora](install-fedora.md).
 
-The following example shows the steps for installing the x64 binary archive. You must choose the
-correct binary archive that matches the processor type for your platform.
+## Red Hat Enterprise Linux (RHEL)
 
-- powershell-7.1.4-linux-arm32.tar.gz
-- powershell-7.1.4-linux-arm64.tar.gz
-- powershell-7.1.4-linux-x64.tar.gz
+RHEL 7 uses yum and RHEL 8 uses the dnf package manager.
 
-#### Linux
+[!INCLUDE [RHEL support](../../includes/rhel-support.md)]
 
-```sh
-# Download the powershell '.tar.gz' archive
-curl -L -o /tmp/powershell.tar.gz https://github.com/PowerShell/PowerShell/releases/download/v7.1.4/powershell-7.1.4-linux-x64.tar.gz
+For more information, see [Install PowerShell on RHEL](install-rhel.md).
 
-# Create the target folder where powershell will be placed
-sudo mkdir -p /opt/microsoft/powershell/7
+## Ubuntu
 
-# Expand powershell to the target folder
-sudo tar zxf /tmp/powershell.tar.gz -C /opt/microsoft/powershell/7
+Ubuntu uses APT (Advanced Package Tool) as a package manager.
 
-# Set execute permissions
-sudo chmod +x /opt/microsoft/powershell/7/pwsh
+[!INCLUDE [Ubuntu support](../../includes/ubuntu-support.md)]
 
-# Create the symbolic link that points to pwsh
-sudo ln -s /opt/microsoft/powershell/7/pwsh /usr/bin/pwsh
-```
+For more information, see [Install PowerShell on Ubuntu](install-ubuntu.md).
 
-### Uninstalling binary archives
+## Raspberry Pi OS
 
-```sh
-sudo rm -rf /usr/bin/pwsh /opt/microsoft/powershell
-```
+[Raspberry Pi OS][raspbian] (formerly Raspbian) is a free operating system based on Debian.
```

Comment:
Shouldn't there be an include for rasbian like the others?

Reply:
No. They only support the current release. No versions to track.